packages/teleterm/src/mainProcess/runtimeSettings.ts

```diff
@@ -14,7 +14,7 @@ const RESOURCES_PATH = app.isPackaged
   ? process.resourcesPath
   : path.join(__dirname, '../../../../');
 
-const TSH_BIN_ENV_VAR = 'TELETERM_TSH_PATH';
```

Comment:
This env var is used in development mode, I just updated it to `CONNECT_TSH_BIN_PATH` so that both modes use the same env var name.